nixos/modules/services/web-servers/nginx/default.nix

```
@@ -422,6 +469,15 @@ in
         '';
       };
 
+      recommendedBrotliSettings = mkOption {
+        default = false;
+        type = types.bool;
+        description = lib.mdDoc ''
+          Enable recommended brotli settings. Required add the brotli module to
+          services.nginx.additionalModules.
```

```suggestion
          Enable recommended brotli settings. Learn more about compression in Brotli format [here](https://github.com/google/ngx_brotli/blob/master/README.md).
          
          Requires adding the brotli module to `services.nginx.additionalModules`:
          
          ```nix
          services.nginx = {
            additionalModules = [ pkgs.nginxModules.brotli ];
            recommendedBrotliSettings = true;
          };
          ```
```

---

Thanks, fixed.